src/_data/capability-matrix.yml

```diff
@@ -34,6 +36,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ParDo applies per-element transformations as Spark FlatMapFunction.
+          - class: apex
+            l1: 'Yes'
+            l2: fully supported
+            l3: ParDo supported through Apex operator that wraps the function.
```

Comment:
Is there anything you want to say about the implementation, in case a user is reading it for details? For example, how DataflowRunner discusses bundle sizes and SparkRunner says exactly what the underlying mechanism is.

IMO all of this is optional - we only really need a comment if there is some limitation or pitfall.